packages/studio-base/webpack.ts

```diff
@@ -49,7 +51,11 @@ export function makeConfig(
   const isDev = argv.mode === "development";
   const isServe = argv.env?.WEBPACK_SERVE ?? false;
 
-  const { allowUnusedVariables = isDev && isServe, version } = options;
+  const {
+    allowUnusedVariables = isDev && isServe,
+    version,
+    storybookConfigPath = undefined,
```

Comment:
nit: undefined will already be the default
```suggestion
    storybookConfigPath,
```